internal/featuretoggle/monitoring_stack_feature_toggle.go

```diff
@@ -36,128 +25,8 @@ func (m MonitoringStackFeatureToggle) Name() string {
 	return "Monitoring Stack Reconciliation Feature Toggle"
 }
 
-func (m MonitoringStackFeatureToggle) IsEnabled() bool {
-	return m.FeatureTogglesInCluster.ExperimentalFeatures
-}
-
-func (m MonitoringStackFeatureToggle) IsEnabledOnTestEnv() bool {
-	enabled, ok := os.LookupEnv("EXPERIMENTAL_FEATURES")
-	return ok && enabled == "true"
-}
-
-func (m *MonitoringStackFeatureToggle) Enable(ctx context.Context) error {
-	adoInCluster := addonsv1alpha1.AddonOperator{}
-	if err := m.Client.Get(ctx, types.NamespacedName{Name: addonsv1alpha1.DefaultAddonOperatorName}, &adoInCluster); err != nil {
-		if errors.IsNotFound(err) {
-			adoObject := addonsv1alpha1.AddonOperator{
-				ObjectMeta: metav1.ObjectMeta{
-					Name: addonsv1alpha1.DefaultAddonOperatorName,
-				},
-				Spec: addonsv1alpha1.AddonOperatorSpec{
-					FeatureToggles: addonsv1alpha1.AddonOperatorFeatureToggles{
-						ExperimentalFeatures: true,
-					},
-				},
-			}
-			if err := m.Client.Create(ctx, &adoObject); err != nil {
-				return err
-			}
-			m.FeatureTogglesInCluster = adoObject.Spec.FeatureToggles
-			return nil
-		}
-		return err
-	}
-	// no need to do anything if its already enabled
-	if adoInCluster.Spec.FeatureToggles.ExperimentalFeatures {
-		return nil
-	}
-	adoInCluster.Spec.FeatureToggles.ExperimentalFeatures = true
-	if err := m.Client.Update(ctx, &adoInCluster); err != nil {
-		return fmt.Errorf("failed to enable the feature toggle in the AddonOperator object: %w", err)
-	}
-	m.FeatureTogglesInCluster = adoInCluster.Spec.FeatureToggles
-	return nil
-}
-
-func (m *MonitoringStackFeatureToggle) PreClusterCreationSetup(ctx context.Context) error {
-	return nil
-}
-
-func renderObservabilityOperatorCatalogSource(ctx context.Context, cluster *dev.Cluster) (*operatorsv1alpha1.CatalogSource, error) {
-	objs, err := dev.LoadKubernetesObjectsFromFile("config/deploy/observability-operator/catalog-source.yaml.tpl")
-	if err != nil {
-		return nil, fmt.Errorf("failed to load the prometheus-remote-storage-mock deployment.yaml.tpl: %w", err)
-	}
-
-	// Replace version
-	observabilityOperatorCatalogSource := &operatorsv1alpha1.CatalogSource{}
-	if err := cluster.Scheme.Convert(&objs[0], observabilityOperatorCatalogSource, ctx); err != nil {
-		return nil, fmt.Errorf("failed to convert the catalog source: %w", err)
-	}
-
-	observabilityOperatorCatalogSourceImage := fmt.Sprintf("quay.io/rhobs/observability-operator-catalog:%s", observabilityOperatorVersion)
-	observabilityOperatorCatalogSource.Spec.Image = observabilityOperatorCatalogSourceImage
-
-	return observabilityOperatorCatalogSource, nil
-}
-
-func (m *MonitoringStackFeatureToggle) PostClusterCreationSetup(ctx context.Context, clusterCreated *dev.Cluster) error {
```

Comment:
This is still needed right?

Reply:
Yep, I moved it to a new file called `monitoring_stack_feature_toggle_testable.go` This file is intended to hold the implementation of all the methods for the interface `TestableFeatureToggleHandler`